elements/rh-alert/demo/alternate.html

```diff
@@ -0,0 +1,48 @@
+<rh-context-provider color-palette="lightest">
+  <h2>Inline, alternate</h2>
+
+  <rh-alert variant="alternate">
+    <h3 slot="header">Default</h3>
+    <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nullam eleifend elit sed est
+      egestas, a sollicitudin mauris tincidunt.</p>
+    <button slot="actions" data-action="dismiss">Dismiss</button>
+    <button slot="actions" data-action="confirm">Confirm</button>
+  </rh-alert>
+
+  <rh-alert state="info" variant="alternate">
+    <h3 slot="header">Info</h3>
+    <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nullam eleifend elit sed est
+      egestas, a sollicitudin mauris tincidunt.</p>
+    <button slot="actions" data-action="dismiss">Dismiss</button>
+    <button slot="actions" data-action="confirm">Confirm</button>
+  </rh-alert>
+
+  <rh-alert state="success" variant="alternate">
+    <h3 slot="header">Success</h3>
+    <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nullam eleifend elit sed est
+      egestas, a sollicitudin mauris tincidunt.</p>
+    <button slot="actions" data-action="dismiss">Dismiss</button>
+    <button slot="actions" data-action="confirm">Confirm</button>
+  </rh-alert>
+
+  <rh-alert state="warning" variant="alternate">
+    <h3 slot="header">Warning</h3>
+    <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nullam eleifend elit sed est
+      egestas, a sollicitudin mauris tincidunt.</p>
+    <button slot="actions" data-action="dismiss">Dismiss</button>
+    <button slot="actions" data-action="confirm">Confirm</button>
+  </rh-alert>
+
+  <rh-alert state="danger" variant="alternate">
+    <h3 slot="header">Danger</h3>
+    <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nullam eleifend elit sed est
+      egestas, a sollicitudin mauris tincidunt.</p>
+    <button slot="actions" data-action="dismiss">Dismiss</button>
+    <button slot="actions" data-action="confirm">Confirm</button>
+  </rh-alert>
+</rh-context-provider>
+
+<script type="module">
+  import '@rhds/elements/rh-alert/rh-alert.js';
+  import '@rhds/elements/lib/elements/rh-context-provider.js';
```

Comment:
```Suggestion
  import '@rhds/elements/rh-context-provider/rh-context-provider.js';
```

Sub page demo's are broken on the DP.  I have a feeling this is preprep for https://github.com/RedHat-UX/red-hat-design-system/pull/996 but should we wait to fix after that merges so we aren't pushing out broken demos?

Reply:
yeah might as well wait on that pr